src/EditorFeatures/CSharpTest/Formatting/CodeCleanupTests.cs

```diff
@@ -627,20 +627,14 @@ private void Method()
     }
 
     [Theory]
-    [InlineData(LanguageNames.CSharp, 51)]
-    [InlineData(LanguageNames.VisualBasic, 88)]
-    public void VerifyAllCodeStyleFixersAreSupportedByCodeCleanup(string language, int numberOfUnsupportedDiagnosticIds)
+    [InlineData(LanguageNames.CSharp)]
+    [InlineData(LanguageNames.VisualBasic)]
+    public void VerifyAllCodeStyleFixersAreSupportedByCodeCleanup(string language)
     {
         var supportedDiagnostics = GetSupportedDiagnosticIdsForCodeCleanupService(language);
 
         // No Duplicates
         Assert.Equal(supportedDiagnostics, supportedDiagnostics.Distinct());
-
-        // Exact Number of Unsupported Diagnostic Ids
-        var ideDiagnosticIds = typeof(IDEDiagnosticIds).GetFields().Select(f => f.GetValue(f) as string).ToArray();
-        var unsupportedDiagnosticIds = ideDiagnosticIds.Except(supportedDiagnostics).ToArray();
-
-        Assert.Equal(numberOfUnsupportedDiagnosticIds, unsupportedDiagnosticIds.Length);
```

Comment:
this part of hte test has never been helpful.  and every time we add an analyzer/fixer it fails and all we do is increment those numbers.  so i'm deleting.